content/3_Silver/Prefix_Sums.mdx

```diff
@@ -681,6 +690,371 @@ as expected.
 
 Since no matter the size of the submatrix we are summing, we only need to access four values of the 2D prefix sum array, this runs in $\mathcal{O}(1)$ per query after an $\mathcal{O}(NM)$ preprocessing.
 
+## Creation From Endpoints
+
+<FocusProblem problem={problems.sample3[0]} />
+
+### 1-Dimensional
+
+A common application of prefix sums is to determine, given a set of intervals, the number of intervals that cover any particular point.
+
+In the focus problem, we are given $n$ intervals $[a,b]$, and we want to find the maximum number of intervals that cover any particular point.
+
+An naïve approach to this problem is to compute the intervals that cover each point by looping through each index from $[a,b]$ individually and incrementing by 1, resulting in an $\mathcal{O}(NQ)$ time complexity (where N is the maximum size of our interval and Q is the number of queries), which easily times out (think what happens when the interval [0, N] is queried many times).
```

Comment:
I'd prefer more clarifications here.

Maybe "... problem is to create an array $\texttt{ctr} where $\texttt{ctr}[i]$ is the number of intervals that contain point $i$." We can do this by taking each interval $[a, b]$ and increasing $\texttt{ctr}[i]$ by $1$ for each index in $a \leq i \leq b$"